src/Core/Localization/Pack/Loader/RemoteLocalizationPackLoader.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Localization\Pack\Loader;
+
+use PrestaShop\PrestaShop\Core\ConfigurationInterface;
+
+/**
+ * Class RemoteLocalizationPackLoader is responsible for loading localization pack data from prestashop.com
+ */
+final class RemoteLocalizationPackLoader extends AbstractLocalizationPackLoader
+{
+    /**
+     * @var ConfigurationInterface
+     */
+    private $configuration;
+
+    /**
+     * @param ConfigurationInterface $configuration
+     */
+    public function __construct(ConfigurationInterface $configuration)
+    {
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLocalizationPackList()
+    {
+        $apiUrl = $this->configuration->get('_PS_API_URL_');
+
+        $xmlLocalizationPacks = $this->loadXml($apiUrl.'/rss/localization.xml');
+        if (!$xmlLocalizationPacks) {
+            return null;
+        }
+
+        return $xmlLocalizationPacks;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLocalizationPack($countryIso)
+    {
+        $psVersion = str_replace('.', '', $this->configuration->get('_PS_VERSION_'));
+        $psVersion = substr($psVersion, 0, 2);
```

Comment:
Hum, what is the result if we are in version `1.12.1.0`?

Reply:
well, it definitely wouldnt work.
however, localization API now supports 2 digits only (16 for 1.6.x, 17 for 1.7.x) and 1.10+ Prestashop is not coming any time soon (or maybe even never :smiley: )  so i think we should keep same logic as it was in legacy code. what do you think?

Reply:
@eternoendless @mickaelandrieu Wdty? Could be a surprise bug in the future :/

Reply:
I have the feeling that someone has already done that... isn't @michaelKaefer? :)

Reply:
not really, in this case `http://api.prestashop.com/localization/{psVersion}/{countryIso}.xml` is used to get localization pack data. `{psVersion}` is passed as `17` and not as `1.7` (same for 1.6, it goes as `16` value).
In legacy part it used to take first 2 string letters from version, so i kept it that way, should we change it?

Reply:
sure! In `Core/Foundation/Version` class (you may need to create it).

I'll complete an issue in prestafony project with some hints to help you, there is no hurry here ;)

Reply:
@sarjon, this task is not required in this contribution I consider your work done after a final rebase (I've just merged a migration, so the Link class needs to be updated).

Reply:
@mickaelandrieu Ok, thanks! I'll write you :+1: